docs/pages/deployment/tls-configuration.rst

```diff
@@ -145,58 +145,60 @@ For `NGINX <https://www.nginx.com/>`_ the proxy configuration could look as foll
 
 .. code-block::
 
-    upstream nuts-node-http-public {
-      server nuts-node:80;
-    }
-    upstream nuts-node-http-n2n {
-      server nuts-node:8080;
-    }
-    upstream nuts-node-grpc {
-      server nuts-node:5555;
-    }
-
-    server {
-      server_name nuts-grpc;
-      listen                    5555 ssl http2;
-      ssl_certificate           /etc/nginx/ssl/server.pem;
-      ssl_certificate_key       /etc/nginx/ssl/key.pem;
-      ssl_client_certificate    /etc/nginx/ssl/truststore.pem;
-      ssl_verify_client         on;
-      ssl_verify_depth          1;
-
-      location / {
-        grpc_pass grpc://nuts-node-grpc;
-        grpc_set_header X-SSL-CERT $ssl_client_escaped_cert;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for; // for correct IP logging
-      }
+    events {
+        worker_connections  1024;
```

Comment:
not sure if we should include this. I think it's default? But there's more defaults, like log files. We should probably only show the stuff here that's relevant for TLS configuration? Otherwise it can get quite elaborate.

Reply:
I got an error message that the events section was missing when leaving this out.

Probably fine to leave it out though.